src/pages/Expenses.jsx

```diff
@@ -0,0 +1,19 @@
+import { Box, Flex } from '@chakra-ui/react';
+import i18next from 'i18next';
+
+import { Footer } from '@/components';
+
+export const Expenses = () => {
+  return (
+    <>
+      <Box bg="orange.300" w="100%" p={4} color="cornsilk">
```

Comment:
Please adhere to the theme and color mode

Reply:
Done